impeller/scene/importer/BUILD.gn

```diff
@@ -85,9 +88,24 @@ impeller_component("scenec") {
 
   sources = [ "scenec_main.cc" ]
 
-  deps = [ ":scenec_lib" ]
+  deps = [ ":importer_lib" ]
```

Comment:
Is this based off of an old commit? I thought scenec was the name we settled on?

Reply:
I figured it would make sense to call the frontend/executable name "scenec" and the namespace/library with the importing utilities "importer", similar to how "impellerc" is the frontend for the "compiler" lib. Perhaps I should use the `scenec` name all around?